### ISSUE TITLE
Naming/PredicateMethod still reporting issues even if added to rubocop_todo.yml

### ISSUE THREAD

**Issue by julioag-rmd** [rubocop/rubocop#14556]
## Expected behavior

```gherkin
Given I have enabled Naming/PredicateMethod in my `.rubocop.yml`
And I have run `bundle exec rubocop --regenerate-todo`
And some instances of Naming/PredicateMethod have been added to the to-do
And I run `bundle exec rubocop`
Then I should NOT see those instances of Naming/PredicateMethod as errors or warnings
```

## Actual behavior

When I run `bundle exec rubocop`
I get errors about `Naming/PredicateMethod` that are listed in my `.rubocop_todo.yml`

## Steps to reproduce the problem

Have a class with a method that returns true/false

```rb
def can_do_review
  true
end
```

Enable the cop with this config:

```rb
Naming/PredicateMethod:
  Enabled: true
```

Run `bundle exec rubocop`, see the issue
Run `bundle exec rubocop --regenerate-todo`
Run `bundle exec rubocop`, keep seeing the issue (‼️)

## RuboCop version

```
➜ bundle exec rubocop -V
1.80.2 (using Parser 3.3.9.0, Prism 1.5.1, rubocop-ast 1.47.1, analyzing as Ruby 3.4, running on ruby 3.4.5) [arm64-darwin24]
  - rubocop-graphql 1.5.5
  - rubocop-performance 1.26.0
  - rubocop-rails 2.33.3
  - rubocop-rspec 3.7.0
  - rubocop-rspec_rails 2.31.0
  - rubocop-capybara 2.22.1
  - rubocop-factory_bot 2.27.1
```

**Comment by Earlopain** [comment]
Do you happen to have some Excludes already in your own `.rubocop.yml`? Unfortunatly, rubocop does _not_ merge them, even if coming from its own todo file. See https://github.com/rubocop/rubocop/issues/9325

With the steps you've given, it works as expected for me.

**Comment by julioag-rmd** [comment]
Hi @Earlopain thanks for the response!

Well, yes, we have several "global" excludes and particularly an exclude on this specific cop.

What is strange is that the errors I was seeing were 
1) already in the to-do
2) in files NOT affected by any of the excludes

**Comment by Earlopain** [comment]
This is "intended". You have to apply the workaround from the issue I linked to in my previous response.

**Comment by julioag-rmd** [comment]
Thanks @Earlopain !!! 

We ended up fixing all existing instances of violations of the cop, so I'm not sure this can be closed...
